### DISCUSSION TITLE
Δηλώσεις για τις παρουσιάσεις προόδου ομαδικής ΓΕ3 Μέρος Δ

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#60] [Show and tell]
Στα παρακάτω σχόλια μπορείτε να κάνετε δήλωση πρόθεσης της προαιρετικής παρουσίασης ομαδικής εργασίας με την ανάρτηση τουλάχιστον μια μέρα πριν την προθεσμία, 

Οι παρουσιάσεις θα γίνουν με σειρά προτεραιότητας ανάλογα με τον χρόνο δήλωσης παρακάτω.

Σε περίπτωση που ο χρόνος της ΟΣΣ δεν είναι αρκετός τότε κάποιες παρουσιάσεις μπορεί να γίνουν σε επόμενη συμπληρωματική ΟΣΣ

**Comment by afoscolou** [comment]
Η ομάδα μας θα ήθελε να παρουσιάσει το μέρος Δ της εργασίας 3 το Σάββατο 4/2/2023 για να έχουμε προχωρήσει και στα ερωτήματα της αξιολόγησης. Παρακαλούμε να μας επιβεβαιώσετε ότι θα γίνει συμπληρωματική ΟΣΣ και εκείνο το Σάββατο.

**Reply by epidrome** [reply]
ναι, σίγουρα μπορούμε να κάνουμε μια ακόμη μικρή συμπληρωματική ΟΣΣ3

**Reply by afoscolou** [reply]
Ευχαριστούμε πολύ. Δηλώνουμε ότι θα κάνουμε την παρουσίαση το Σάββατο 4/2/2023.

**Reply by sotkot** [reply]
Καλησπέρα,

Επισυνάπτω την παρουσίαση της ομαδικής μας εργασίας σε pdf (Μέρος Δ):
[SDY60_3ERG_ANDREADAKIS_KOTROTSIOS_MIOGLOU_FOSCOLOU_part_D.pdf](https://github.com/sdy60/syllabus/files/10580842/SDY60_3ERG_ANDREADAKIS_KOTROTSIOS_MIOGLOU_FOSCOLOU_part_D.pdf)

**Comment by gregtsag** [comment]
[PRES GE2_1 KAM_2KAR_TSA -.pptx](https://github.com/sdy60/syllabus/files/10521137/PRES.GE2_1.KAM_2KAR_TSA.-.pptx)

Καλησπέρα, η παρουσίαση για την ΟΣΣ στις 28/1. Ομάδα Καμπορούπολος, Καρυπίδου, Καραγιαννάκης, Τσαγρής

**Comment by ghost** [comment]
Λόγω προβλήματος που υπάρχει με ένα μέλος της ομάδας μας, δε θα μπορέσουμε να παρουσιάσουμε σήμερα. Προσπαθώ να το επικοινωνήσω μόλις το έμαθα σε περίπτωση που κρίνεται ότι δεν υπάρχει λόγος για συνάντηση σήμερα αφού και η άλλη ομάδα δε θα παρουσιάσει. Ελπίζω να μας δωθεί η ευκαιρία να παρουσιάσουμε το επόμενο Σάββατο.
Ευχαριστώ

**Reply by afoscolou** [reply]
Νομίζω ότι δεν υπάρχει πρόβλημα για την αναβολή της συνάντησης και όλοι θα συμφωνήσουν. Αν κάποιος έχει κάποια απορία μπορεί να τη λύσει ασύγχρονα. 
Παρεμπιπτόντως σας έχω ρωτήσει κάτι κύριε Χωριανόπουλε μέσω email. Απαντήστε μου μόλις μπορέσετε.
